cmd/tdex/market.go

```diff
@@ -690,3 +717,52 @@ func marketUpdatePriceAction(ctx *cli.Context) error {
 	fmt.Println("price has been updated")
 	return nil
 }
+
+func marketGetReport(ctx *cli.Context) error {
```

Comment:
just to stick with the convetion naming adopted for cli's actions:

```suggestion
func marketReportAction(ctx *cli.Context) error {
```